### ISSUE TITLE
Does not create nested directories

### ISSUE THREAD

**Issue by IanVS** [glenjamin/mocha-multi#15]
It looks like https://github.com/glenjamin/mocha-multi/pull/13 added support for creating a directory for the reporter output, but it does not handle nested directories (`mkdir` needs a `-p` option).

Here's the stack trace I get when trying to create `build/reports/xml/mocha-results.xml` when only `build` exists.

```
fs.js:842
  return binding.mkdir(pathModule._makeLong(path),
                 ^

Error: ENOENT: no such file or directory, mkdir 'build/reports/xml'
    at Error (native)
    at Object.fs.mkdirSync (fs.js:842:18)
    at resolveStream (/Users/ianvs/code/nutshell/nutshell-client/node_modules/mocha-multi/mocha-multi.js:145:8)
    at /Users/ianvs/code/nutshell/nutshell-client/node_modules/mocha-multi/mocha-multi.js:78:18
    at Array.map (native)
    at initReportersAndStreams (/Users/ianvs/code/nutshell/nutshell-client/node_modules/mocha-multi/mocha-multi.js:73:16)
    at new MochaMulti (/Users/ianvs/code/nutshell/nutshell-client/node_modules/mocha-multi/mocha-multi.js:29:17)
    at Mocha.run (/Users/ianvs/code/nutshell/nutshell-client/node_modules/mocha/lib/mocha.js:474:18)
    at Object.<anonymous> (/Users/ianvs/code/nutshell/nutshell-client/node_modules/mocha/bin/_mocha:403:18)
    at Module._compile (module.js:398:26)
    at Object.Module._extensions..js (module.js:405:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Function.Module.runMain (module.js:430:10)
    at startup (node.js:141:18)
    at node.js:980:3
```

**Comment by glenjamin** [comment]
Fixed in 20d1327 - will be released in `0.9.0`

**Comment by IanVS** [comment]
:+1: Thanks.
